test/system/test_cvp_client_api.py

```diff
@@ -2127,8 +2131,7 @@ def test_api_inventory(self):
                                                       test_dev, orig_configlets,
                                                       create_task=True)
         # execute returned task and wait for it to complete
-        task_res = self.api.execute_task(results['data']['taskIds'][0])
-        self.assertEqual(task_res, None)
+        self.api.execute_task(results['data']['taskIds'][0])
```

Comment:
This is removed because execute_task doesn't actually have a return statement and hence the linter was complaining. I can look into adding a return statement for execute_take as an alternative.